### PR TITLE
Add Vaadin extension to ecosystem CI

### PR DESCRIPTION
**Pull request by mcollovati**

### CODE REVIEW

**Comment by geoand** [comment]
I think it would be best to have the issue in `https://github.com/quarkiverse/`.

**Comment by mcollovati** [comment]
@geoand In our extension repository we have the main branch for Quarkus 3 and one for Quarkus 2.
Should we open two issues, one per issue template?

**Comment by geoand** [comment]
@gastaldi is better situated to answer that one :)

**Comment by gastaldi** [comment]
> In our extension repository we have the main branch for Quarkus 3 and one for Quarkus 2. Should we open two issues, one per issue template?

Yes, also for tracking purposes please create these issues in your own repository (https://github.com/vaadin/quarkus) with an `ECOSYSTEM_CI_TOKEN` secret (see https://github.com/quarkusio/quarkus-ecosystem-ci#what-if-my-repository-is-not-in-the-quarkiverse-organization).

File: vaadin/info.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+url: https://github.com/vaadin/quarkus
+issues:
+  repo: quarkusio/quarkus
+  latestCommit: 32985
```
Comment (gastaldi) [inline vaadin/info.yaml:4]: To build the 2.x branch, you need to set a new job (in the same [workflow](https://github.com/vaadin/quarkus/blob/master/.github/workflows/quarkus-snapshot.yaml)) and specify the environment variable `ALTERNATIVE="2.x"` for example 

```suggestion
  latestCommit: 32985
alternatives:
    2.x:
      issue: <ISSUE_NUMBER_IN_YOUR_REPO>
      quarkusBranch: "2.16"
```

File: vaadin/info.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+url: https://github.com/vaadin/quarkus
+issues:
+  repo: quarkusio/quarkus
```
Comment (gastaldi) [inline vaadin/info.yaml:3]: Create the tracking issue in your own repository (see https://github.com/quarkusio/quarkus-ecosystem-ci#what-if-my-repository-is-not-in-the-quarkiverse-organization): 
```suggestion
  repo: vaadin/quarkus
```

**Comment by mcollovati** [comment]
Thank you. Just a question: in the example, the issues seem to have been created on `quarkusio/quarkus` repository.
Is this because the policy changed after that one integration was set up?

**Comment by mcollovati** [comment]
I'm asking just to be sure because from the docs I did not get that the issues should be created on the extension repository.

**Review by gastaldi — APPROVED** [review]

**Pull request merged**